pkg/config/manager.go

```diff
@@ -14,6 +14,17 @@ import (
 	"golang.org/x/oauth2/google"
 )
 
+type PostgresConnection struct {
+	Name         string `yaml:"name" json:"name"`
+	Username     string `yaml:"username" json:"username"`
+	Password     string `yaml:"password" json:"password"`
+	Host         string `yaml:"host" json:"host"`
+	Port         int    `yaml:"port" json:"port"`
+	Database     string `yaml:"database" json:"database"`
+	PoolMaxConns int    `yaml:"pool_max_conns" json:"pool_max_conns" default:"10"`
+	SslMode      string `yaml:"ssl_mode" json:"ssl_mode" default:"disable"`
```

Comment:
do we need json counterparts for these? I think it's just YAML

Reply:
Well I'm not sure. The other config types don't have json annotations but they have a `MarshalJSON` [method](https://github.com/bruin-data/bruin/blob/ff74b51e052cf6d0bbd9589af07a61316c9c7be7/pkg/config/manager.go#L72) so I thought it was  necessary?